src/main/java/com/openlattice/pods/ConductorServicesPod.java

```diff
@@ -289,10 +292,15 @@ public OrganizationAssembliesInitializerTask organizationAssembliesInitializerTa
     }
 
     @Bean
-    public EntitySetViewsInitializerTask entityViewsInitializerTask(){
+    public EntitySetViewsInitializerTask entityViewsInitializerTask() {
         return new EntitySetViewsInitializerTask();
     }
 
+    @Bean
+    public AuditInitializationTask auditInitializationTask() {
+        return new AuditInitializationTask( hazelcastInstance );
```

Comment:
Is this task conductor specific or should be moved into AuditingConfigurationPod

Reply:
nope! I'll move it

Reply:
actually -- do you mean only move that one task, or move the task and the dependencies?

Reply:
I ended up leaving it here -- moving it would have required also adding basically all the conductor beans and a bunch of other initialization tasks to AuditingConfigurationPod

Reply:
Why? It should be just moving this one Task bean. Why would it require moving the other stuff? The Task Service will handle making sure dependencies are there.